sdk/monitor/azure-monitor-opentelemetry-exporter/samples/traces/sample_cosmos.py

```diff
@@ -0,0 +1,54 @@
+"""
+Examples to show usage of the azure-core-tracing-opentelemetry
+with the storage SDK and exporting to Azure monitor backend.
+This example traces calls for creating a container using storage SDK.
+The telemetry will be collected automatically and sent to Application
+Insights via the AzureMonitorTraceExporter
+"""
+
+import os
+
+# Declare OpenTelemetry as enabled tracing plugin for Azure SDKs
+from azure.core.settings import settings
+from azure.core.tracing.ext.opentelemetry_span import OpenTelemetrySpan
+
+settings.tracing_implementation = OpenTelemetrySpan
+
+# Regular open telemetry usage from here, see https://github.com/open-telemetry/opentelemetry-python
+# for details
+from opentelemetry import trace
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+
+trace.set_tracer_provider(TracerProvider())
+tracer = trace.get_tracer(__name__)
+
+# azure monitor trace exporter to send telemetry to appinsights
+from azure.monitor.opentelemetry.exporter import AzureMonitorTraceExporter
+span_processor = BatchSpanProcessor(
+    AzureMonitorTraceExporter.from_connection_string(
+        os.environ["APPLICATIONINSIGHTS_CONNECTION_STRING"]
+    )
+)
+trace.get_tracer_provider().add_span_processor(span_processor)
+
+# Example with CosmosDB SDKs
+from azure.cosmos import exceptions, CosmosClient, PartitionKey
+
+url = os.environ["ACCOUNT_URI"]
+key = os.environ["ACCOUNT_KEY"]
```

Comment:
let's be a little explicit with the variable names - ACCOUNT_URL and key are too generic

Reply:
This is from cosmosdb [docs](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/cosmos/azure-cosmos#create-the-client).